### ISSUE TITLE
508 Defect | Screenreader | Covid-19 Breadcrumb

### ISSUE THREAD

**Issue by SKasimow** [department-of-veterans-affairs/va.gov-team#29575]
Accessibility issue discovered:  when testing on Windows/NVDA/Chrome, "Vacunas contra COVID-19 en el VA" in the breadcrumb is missing a lang="es" tag.
